CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ Bug Fixes
 
 - `#623 <https://github.com/pytest-dev/pytest-xdist/issues/623>`_: Gracefully handle the pending deprecation of Node.fspath by using config.rootpath for topdir.
 
+Features
```

Comment:
Sorry for not being clearer, you should actually create a file `changelog/650.feature.rst`, just with this contents:

```rst
Added new ``pytest_handlecrashitem`` hook to allow handling and rescheduling crashed items.
```

We use [towncrier](https://pypi.org/project/towncrier/) to manage the changelog, which is automatically generated before each release.

Reply:
Went ahead and did it myself. 👍